projects/storefrontlib/src/shared/components/card/card.component.html

```diff
@@ -83,13 +75,23 @@ <h4 *ngIf="content.title" class="cx-card-title">
             tabindex="0"
             >{{ action.name }}</a
           >
-          <a
-            *ngSwitchDefault
-            href="{{ action.link }}"
-            class="card-link btn-link cx-action-link"
-            tabindex="0"
-            >{{ action.name }}</a
-          >
+          <div *ngIf="!content.header">
```

Comment:
`<div *ngIf="!content.header">`

Is this the logic to know if the address is already selected?